docs/developers_guide/landice/test_groups/mesh_modifications.rst

```diff
@@ -0,0 +1,58 @@
+.. _mesh_modifications:
```

Comment:
@matthewhoffman, this should have been named `dev_mesh_modifications`.  It has the same name as the section in the user's guide, which is producing warnings when building the documentation.  Could you open a small PR to fix this when you have time?

(I accidentally commented previously on the commit, not the PR.)

Reply:
Addressed in #808